common4j/src/main/com/microsoft/identity/common/java/util/ported/LocalBroadcaster.java

```diff
@@ -82,6 +82,13 @@ public void run() {
         });
     }
 
+    /**
+     * Clears the receivers associated with this instance.
+     */
+    public void clearReceivers() {
```

Comment:
Can you please also add tests for this and the previously added methods.
Thanks

Reply:
done